controller/applicationsController.js

```diff
@@ -0,0 +1,38 @@
+const Applications = require("../models/applications");
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+
+exports.update = async (req, res, next) => {
+  const { applicationId } = req.params;
+  const { applicationName } = req.body;
+
+  Applications.findById(applicationId)
```

Comment:
This should instead be a `find` instead of `findById` searching for `applicationId` and `organizationId`. This ensures that the application being updated actually belongs to the organization. The `organizationId` is available on `req.token`. This is made possible by the `orgAuthMW` decoding the organizationToken to get this value and attaching it to the request as `req.toke.organizationId`. Please, endeavour to make the change